kingpin/actors/aws/test/integration_cloudformation.py

```diff
@@ -103,8 +104,9 @@ class IntegrationStack(testing.AsyncTestCase):
     * Delete that same stack
 
     Requirements:
-        Your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY must have access to
-        create CF stacks. The stack we create is extremely simple, and should
+        Your must have an AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, and AWS_SESSION_TOKEN
```

Comment:
Nit should say something like: access key id and access key and optionally session token when using an assumed role.

Reply:
Besides these comments, is there anything else blocking?

Reply:
changed the verbiage to:

        You must have an AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY if using
        an IAM user/role and also an AWS_SESSION_TOKEN if using temporary access credentials,
        with permissions to ______